nixos/common/default.nix

```diff
@@ -20,9 +20,6 @@
   # - for containers we currently rely on the `stage-2` init script that sets up our /etc
   boot.initrd.systemd.enable = lib.mkDefault (!config.boot.swraid.enable && !config.boot.isContainer);
 
-  # Work around for https://github.com/NixOS/nixpkgs/issues/124215
-  documentation.info.enable = false;
```

Comment:
This work around is a couple of years old now so I'm assuming that it is safe to remove?